### PR TITLE
Add support for BatchPutPropertyValues

### PR DESCRIPTION
**Pull request by toddtreece**

### CODE REVIEW

File: pkg/plugin/twinmaker/utils.go [hunk 1]
```diff
@@ -353,7 +353,6 @@ func (s *twinMakerHandler) GetComponentHistoryWithLookupHelper(ctx context.Conte
 								}
 							}
 						}
-						break
```
Comment (toddtreece) [inline pkg/plugin/twinmaker/utils.go:356]: why was this removed?

Comment (haweston) [reply pkg/plugin/twinmaker/utils.go:356]: We have multiple alarm component types now.  This break causes a premature return after one is matched leaving the others processed.

File: src/hooks/useTwinMakerClient.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { configureSdkWithDataSource, DataSourceParams } from 'panels/sdkInit';
+import IoTTwinMaker from 'aws-sdk/clients/iottwinmaker';
+
+export const useTwinMakerClient = (
```
Comment (hwandersman) [inline src/hooks/useTwinMakerClient.ts:5]: Is the TwinMaker client actually used anywhere? What's the purpose of this?

Comment (toddtreece) [reply src/hooks/useTwinMakerClient.ts:5]: @haweston is this from the previous iteration?

Comment (haweston) [reply src/hooks/useTwinMakerClient.ts:5]: Alarm panel was using this, but isn't now.

Comment (haweston) [reply src/hooks/useTwinMakerClient.ts:5]: It still calls the hook, but no longer uses it in a meaningful way other than making sure the data source is connected.

Comment (hwandersman) [reply src/hooks/useTwinMakerClient.ts:5]: Let's avoid using the UxSDK altogether here if we can, so we don't use different credentials on the browser. Can probably just check if the datasource is configured - just check the `dataSourceParams`

Comment (haweston) [reply src/hooks/useTwinMakerClient.ts]: Removed

**Review by haweston — APPROVED** [review]

**Review by haweston — COMMENTED** [review]
I think the 3 HDR files in src/static got pulled and a rebase here is readding them?
